libs/core/src/lib/sinks.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { Subject, of } from 'rxjs';
+import { Effect } from './effect';
+import { setup } from './test-helper';
+
+it('sources data from subject', () => {
+  const value = { count$: new Subject() };
+  const effect: Effect<typeof value> = ({ sinks }) => {
+    return of('hello world').pipe(sinks.count$());
+  };
+  setup({ value, effect }, ({ expectObservable, effect$, storeEvent$ }) => {
+    expectObservable(effect$).toBe('(a|)', { a: 'hello world' });
+
+    expectObservable(storeEvent$).toBe('(abcde)', {
```

Comment:
Some overlap here in these tests. In the future we could maybe make these all unit tests, and extract the "integration tests" to live somewhere else & DRY it up. For now, it seemed better to write tests specifically for testing the sources & sinks, but to also test via the top level effects API / test helper for more verification & higher fidelity.